src/models/window.py

```diff
@@ -33,8 +35,11 @@ def clear(self):
 
     def update(self):
         """Updates the contents of the entire window."""
+        # tick the clock to ensure that the game runs at the target FPS
+        delta_time = self._clock.tick(TARGET_FPS) / 16.666666666666668
```

Comment:
tick returns the number of ms since the last frame, which should equal the divisor when running at 60FPS. I did this so that the player's speed doesn't need to be changed (because otherwise it gets multiplied by a rather large amount). In retrospect, it's kind of a weird number, but it lets us use reasonable values for the player's speed.

Reply:
oh well if its in ms we should convert it to seconds, so divide by 1000